front/cypress/e2e/report_builder/widgets/participants_timeline_widget.cy.ts

```diff
@@ -0,0 +1,164 @@
+    const ideaContent = randomString();
+
+    cy.apiCreateProject({
+      title: projectTitle,
+      descriptionPreview: projectDescriptionPreview,
+      description: projectDescription,
+      publicationStatus: 'published',
+    })
+      .then((project) => {
+        projectId = project.body.data.id;
+        projectSlug = project.body.data.attributes.slug;
+
+        return cy.apiCreatePhase({
+          projectId,
+          title: phaseTitle,
+          startAt: moment().subtract(9, 'month').format('DD/MM/YYYY'),
+          participationMethod: 'ideation',
+          canPost: true,
+          canComment: true,
+          canReact: true,
+          allow_anonymous_participation: true,
+        });
+      })
+      .then((phase) => {
+        cy.apiCreateIdea({
+          projectId,
+          ideaTitle,
+          ideaContent,
+          phaseIds: [phase.body.data.id],
+        });
+      })
+      .then(() => {
+        return cy.apiCreatePhase({
+          projectId,
+          title: randomString(),
+          startAt: moment().subtract(29, 'day').format('DD/MM/YYYY'),
+          participationMethod: 'information',
+        });
+      })
+      .then((phase) => {
+        phaseId = phase.body.data.id;
+      });
+  });
+
+  beforeEach(() => {
+    cy.setAdminLoginCookie();
+
+    cy.apiCreateReportBuilder(phaseId).then((report) => {
+      reportId = report.body.data.id;
+      cy.intercept('PATCH', `/web_api/v1/reports/${reportId}`).as(
+        'saveReportLayout'
+      );
+      cy.intercept('GET', `/web_api/v1/reports/${reportId}`).as(
+        'getReportLayout'
+      );
+      cy.visit(`/admin/reporting/report-builder/${reportId}/editor`);
+    });
+  });
+
+  after(() => {
+    cy.apiRemoveProject(projectId);
+  });
+
+  afterEach(() => {
+    cy.apiRemoveReportBuilder(reportId);
+  });
+
+  it('handles Participants timeline widget correctly', function () {
+    cy.get('#e2e-draggable-active-users-widget').dragAndDrop(
+      '#e2e-content-builder-frame',
+      {
+        position: 'inside',
+      }
+    );
+
+    cy.wait(1000);
+
+    // Change widget title
+    cy.get('#e2e-analytics-chart-widget-title')
+      .clear()
+      .type('New Widget Title');
+
+    cy.wait(1000);
+
+    // Set project filter
+    cy.get('#e2e-report-builder-project-filter-box select').select(projectId);
+
+    // Set date range and compare with previous period
+    cy.get('.e2e-statistic-delta').should('not.exist');
+    cy.get('#e2e-start-date-input')
+      .clear({ force: true })
+      .type(moment().subtract(5, 'day').format('DD/MM/YYYY'), { force: true });
+    cy.get('#e2e-content-builder-settings').click();
+
+    cy.wait(1000);
+
+    cy.get('#e2e-compare-previous-period-toggle').parent().click();
+    cy.get('.e2e-statistic-delta').should('exist');
+
+    // Confirms that the widget displays correctly on live report
+    cy.get('#e2e-content-builder-topbar-save').click();
+    cy.wait('@saveReportLayout');
```

Comment:
How come the save button works if the request is intercepted? 🤔

Reply:
It's because right after this command, we navigate to a different page. If you don't wait for the request to be sent, you are already navigating to the next page before the browser has time to send it. 

You'd expect the button click and sending part of the request to be synchronous but apparently it's not 🤷